app/qml/gps/MMStakeoutPanel.qml

```diff
@@ -58,15 +58,26 @@ MMDrawer {
     root.open()
   }
 
+  Item {
+    focus: true
+
+    Keys.onPressed: {
+      if ( event.key === Qt.Key_Back || event.key === Qt.Key_Escape ) {
+        root.endStakeout()
+        event.accepted = true
+      }
+    }
+  }
```

Comment:
This is already implemented in https://github.com/MerginMaps/mobile/blob/master/app/qml/components/MMDrawer.qml#L65

so either that MMDrawer implementation doesn't work and we need to fix it (e.g. move it to separate Item as here) or it is not needed here at all if you connect correctly onBackClicked?